data/mods/Magiclysm/monsters/holiday_magiclysm.json

```diff
@@ -0,0 +1,110 @@
+[
+  {
+    "id": "mon_yulecat",
+    "type": "MONSTER",
+    "name": "Yule Cat",
+    "description": "Magical beast from Icelandic folklore, known for their catlike appearance and their hobby of eating Workshop Elves.",
+    "bodytype": "lion",
+    "default_faction": "magical_beast",
+    "species": [ "MAGICAL_BEAST" ],
+    "volume": "200000 ml",
+    "weight": "780 kg",
+    "hp": 245,
+    "speed": 167,
+    "material": [ "flesh" ],
+    "symbol": "Y",
+    "color": "white",
+    "aggression": 50,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 7,
+    "melee_dice_sides": 8,
+    "melee_cut": 5,
+    "dodge": 5,
+    "armor_bash": 4,
+    "armor_cut": 1,
+    "armor_bullet": 4,
+    "vision_night": 12,
+    "path_settings": { "max_dist": 21 },
+    "harvest": "mammal_fur",
+    "death_function": [ "NORMAL" ],
+    "reproduction": { "baby_monster": "mon_yulecat_cub", "baby_count": 6, "baby_timer": 234 },
+    "baby_flags": [ "WINTER" ],
+    "special_attacks": [ { "type": "bite", "cooldown": 10 }, [ "GRAB", 8 ], [ "scratch", 6 ] ],
+    "anger_triggers": [ "PLAYER_NEAR_BABY" ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "KEENNOSE", "PATH_AVOID_DANGER_2", "WARM", "GRABS" ]
+  },
+  {
+    "id": "mon_yulecat_cub",
+    "type": "MONSTER",
+    "copy-from": "mon_yulecat",
+    "name": { "str": "Yulecat Cub" },
+    "description": "The child of a yulecat, this catlike creature looks as if it poses no threat to you, but looks can be deceiving.",
+    "volume": "120 ml",
+    "weight": "345 g",
+    "hp": 56,
+    "speed": 120,
+    "symbol": "y",
+    "morale": 23,
+    "melee_skill": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 7,
+    "dodge": 6,
+    "harvest": "mammal_fur",
+    "special_attacks": [ [ "EAT_FOOD", 71 ] ],
+    "extend": { "flags": [ "NO_BREED" ], "fear_triggers": [ "PLAYER_CLOSE", "HURT" ] },
+    "delete": { "anger_triggers": "PLAYER_NEAR_BABY" },
+    "upgrades": { "age_grow": 234, "into": "mon_yulecat" }
+  },
+  {
+    "id": "mon_workshop_elf",
+    "type": "MONSTER",
+    "name": { "str": "Workshop Elf", "str_pl": "Workshop Elves" },
+    "description": "Magical humanoids who are known for working in workshops, wearing green, and being proficient at making toys.",
+    "default_faction": "elves",
```

Comment:
Game does not load and complains:
```
 DEBUG    : Monster type 'mon_workshop_elf' has invalid default_faction: 'elves'. Add this faction to json as MONSTER_FACTION type.

 FUNCTION : void MonsterGenerator::finalize_mtypes()
 FILE     : src/monstergenerator.cpp
 LINE     : 373
```

Reply:
I did actually create monster factions, they were accidentally deleted while fixing merge conflicts, I'll add them back, thanks for notifying me.